packages/angular/projects/ui-angular/ng-package.json

```diff
@@ -1,13 +1,20 @@
 {
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/ui-angular",
+  "assets": ["./theme.css"],
   "lib": {
     "entryFile": "src/index.ts",
     "umdModuleIds": {
       "xstate": "xstate",
       "aws-amplify": "aws-amplify",
-      "@aws-amplify/core": "@aws-amplify/core",
-      "@aws-amplify/ui-core": "@aws-amplify/ui-core"
+      "@aws-amplify/core": "@aws-amplify/core"
     }
-  }
+  },
+  "allowedNonPeerDependencies": [
```

Comment:
What does this mean or do?  (It looks like we have to restate our dependencies here. What happens if we forget one?)

Reply:
Angular recommends using `peerDependency` [whenever possible](https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md#general-recommendation-use-peerdependencies-whenever-possible). To add non peer-dependencies, they require we add those to this array. If we forget to add one, `ng build` will complain and fail the build.

While I agree with their approach, dependencies like `xstate` should be `dependencies` and shouldn't need to be added by customers, which is why I added it here.